README.md

```diff
@@ -44,6 +44,33 @@ header:
 
 **NOTE**: The full configurations can be found in [the configuration section](#configurations).
 
+#### Using the Action in Fix Mode
+
+By default the action runs License-Eye in check mode, which will raise an error
+if any of the processed files are missing license headers. If `mode` is set to
+`fix`, the action will instead apply the license header to any processed file
+that is missing a license header. The fixed files can then be pushed back to the
+pull request using another GitHub action. For example:
```

Comment:
So, one action would commit and push, then make the CI failing, right? Just confirming. Then maintainers should rerun the CI task to make it passed.

Reply:
@wu-sheng The SkyWalking-Eyes action will edit the files in place. The subsequent "add-and-commit" action will use git to commit the changed files back to the PR. The action I linked to has a couple returns (see [here](https://github.com/EndBug/add-and-commit#outputs)); if you wanted to, you could add a step to the action which errs if the "add-and-commit" action makes a commit. Automating the re-run is not straightforward because GitHub doesn't like workflows triggering other workflows, so the maintainer will probably need to manually re-run the failed workflow. 

In practice I usually don't worry about this becasue there's usually other changes needed to the PR; so when the PR author submits those changes (which will necessarily need to be synched with the automatic commit) they'll re-trigger CI.